content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -0,0 +1,416 @@
+| `plots`          | List of [plot metrics](/doc/command-reference/plots), and optionally, their default configuration (subfields matching the options of `dvc plots modify`), and whether or not this plots file is <abbr>cached</abbr> ( `true` by default). See the `--plots-no-cache` option of `dvc run`. |
+| `frozen`         | Whether or not this stage is frozen from reproduction                                                                                                                                                                                                                                     |
+| `always_changed` | Whether or not this stage is considered as changed by commands such as `dvc status` and `dvc repro`. `false` by default                                                                                                                                                                   |
+| `meta`           | (Optional) arbitrary metadata can be added manually with this field. Any YAML contents is supported. `meta` contents are ignored by DVC, but they can be meaningful for user processes that read or write `.dvc` files directly.                                                          |
+| `desc`           | (Optional) user description for this stage. This doesn't affect any DVC operations.                                                                                                                                                                                                       |
+
+`dvc.yaml` files also support `# comments`.
+
+Note that we maintain a `dvc.yaml`
+[schema](https://github.com/iterative/dvcyaml-schema) that can be used by
+editors like [VSCode](/doc/install/plugins#visual-studio-code) or
+[PyCharm](/doc/install/plugins#pycharmintellij) to enable automatic syntax
+validation and auto-completion.
+
+> See also
+> [How to Merge Conflicts](/doc/user-guide/how-to/merge-conflicts#dvcyaml).
+
+### Output entries
+
+| Field     | Description                                                                                                                                |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `cache`   | Whether or not this file or directory is <abbr>cached</abbr> (`true` by default). See the `--no-commit` option of `dvc add`.               |
+| `persist` | Whether the output file/dir should remain in place while `dvc repro` runs (`false` by default: outputs are deleted when `dvc repro` starts |
+| `desc`    | (Optional) user description for this output. This doesn't affect any DVC operations.                                                       |
+
+## dvc.lock file
```

Comment:
As mentioned earlier, should `dvc.lock` have its own page? It is a separate file type, and this page is long even without it.

Also, should there be some explanation of the relationship between `dvc.lock` and `.dvc` files? Since they contain similar info, I'm not sure the differences between them are clear.

Reply:
It has it's own page in some iteration and it seemed too little for a whole page. Plus it's kind of part of dvc.yaml (meaningless without it).

And as long as we have a decent right-hand ToC nav, length is probably not a big concern. See https://github.com/iterative/dvc.org/issues/2059#issuecomment-760639766

> should there be some explanation of the relationship between dvc.lock and .dvc files? ... not sure the differences between them are clear

There's no direct relationship, they share one of their purposes (mentioned in the 2nd bullet). Please refer to iterative/dvc/issues/3936 for a deeper dive on this situation.

Thanks for all the feedback!